src/app/task/task.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from "@angular/core";
+
+import { TaskDTO } from "../shared/TaskDTO";
+
+@Component({
+	selector: "task",
+	templateUrl: "./task.component.html",
+	styleUrls: ["./task.component.css"]
+})
+export class TaskComponent implements OnInit {
+
+	public taskTest: TaskDTO;
+	public showTaskDetails: boolean = false;
```

Comment:
```suggestion
	public isTaskDetailsShown: boolean = false;
```
prefix boolean properties with is/are for readability